examples/example.krun

```diff
@@ -73,3 +73,9 @@ TEMP_READ_PAUSE = 1
 
 # CPU pinning (off by default)
 #ENABLE_PINNING = False
+
+# Lower and upper bound for acceptable APERF/MPERF ratios
+AMPERF_RATIO_BOUNDS = 0.97, 1.03
+
+# Rough change in APERF per second above which a core is considered busy.
+AMPERF_BUSY_THRESHOLD = 3.4 * 1024 * 1024 * 1024 / 1000  # 3.4 GHz / 1000
```

Comment:
I was aware that for the main experiment config, we need to add these directives, but apparently got mixed up. Yeesh. Sorry about that.